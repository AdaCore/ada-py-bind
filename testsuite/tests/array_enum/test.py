import gen

p = gen.Point()

print p.coords["x"]
print p.coords["y"]
print p.coords["z"]

p.coords["x"] = 12
p.coords["y"] = 13
p.coords["z"] = 14

print p.coords["x"]
print p.coords["y"]
print p.coords["z"]
