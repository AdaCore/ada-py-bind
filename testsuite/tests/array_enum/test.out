0
0
0
12
13
14
